libs/hdf-converters/src/trufflehog-mapper.ts

```diff
@@ -0,0 +1,97 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {BaseConverter, ILookupPath, MappedTransform} from './base-converter';
+
+export class TrufflehogMapper extends BaseConverter {
+  withRaw: boolean;
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion
+    },
+    version: HeimdallToolsVersion,
+    statistics: {},
+    profiles: [
+      {
+        name: {path: 'wrapper[0].SourceName'},
+        title: {path: 'wrapper[0].SourceName'},
+        supports: [],
+        attributes: [],
+        groups: [],
+        status: 'loaded',
+        controls: [
+          {
+            path: 'wrapper',
+            key: 'id',
+            tags: {
+              nist: 'IA-5(7)',
+              cci: ['CCI-004069', 'CCI-000202', 'CCI-000203', 'CCI-002367'],
+              severity: 'medium'
+            },
+            refs: [],
+            source_location: {},
+            title: {
+              transformer: (data: Record<string, unknown>): string => {
+                return (
+                  _.get(data, 'DetectorName') + '_' + _.get(data, 'DecoderName')
+                );
+              }
+            },
+            id: {
+              transformer: (data: Record<string, unknown>): string => {
+                return (
+                  _.get(data, 'DetectorType') + '_' + _.get(data, 'DecoderName')
```

Comment:
The detector name and type are directly correlated.  What is the point of showing different values here.

Reply:
I will just show the detector type as the id instead.